cli/pkg/kctrl/cmd/app/get.go

```diff
@@ -85,7 +84,10 @@ func (o *GetOptions) Run() error {
 			uitable.NewValueString(app.Namespace),
 			uitable.NewValueString(app.Name),
 			uitable.NewValueString(app.Spec.ServiceAccountName),
-			uitable.NewValueString(app.Status.FriendlyDescription),
+			uitable.ValueFmt{
+				V:     uitable.NewValueString(appStatusString(app)),
+				Error: isFailing || app.Spec.Canceled,
```

Comment:
it's interesting that we are using spec section. should we also check on paused?

Reply:
Mainly because an app being paused or cancelled does not affect the conditions and even the friendly message is ambiguous ("Canceled/paused"). For pkgi there are no changes to status at all.
I was initially coloring "Paused" as yellow but the table formatting was not happy about that.

A bit unsure as to if pause should be colored red as well 🤔 but we can definitely make that change.